hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/CompactionCommand.java

```diff
@@ -297,24 +325,24 @@ public String validateCompaction(
         List<Comparable[]> rows = new ArrayList<>();
         res.stream().forEach(r -> {
           Comparable[] row = new Comparable[]{r.getOperation().getFileId(),
-              r.getOperation().getBaseInstantTime(),
-              r.getOperation().getDataFilePath().isPresent() ? r.getOperation().getDataFilePath().get() : "",
-              r.getOperation().getDeltaFilePaths().size(), r.isSuccess(),
-              r.getException().isPresent() ? r.getException().get().getMessage() : ""};
+                  r.getOperation().getBaseInstantTime(),
```

Comment:
Make sure you are using style/intellij-java-google-style.xml as the formatting standard.